client/js/table-model.js

```diff
@@ -16,6 +16,25 @@ class TableModel {
   setValue(location, value) {
     this.data[this._getCellId(location)] = value;
   }
-}
 
+  
+  _getColNums(col) {
+    let nums = [];
+    for (var row = 0; row < this.numRows; row++) {
```

Comment:
Before the old "var"! Change all the vars to either "const" or "let". For loops use "let".